app.rb

```diff
@@ -0,0 +1,107 @@
+require './book'
+require './game'
+require './author'
+require './item'
+require './label'
+require './music_album'
+require './genre'
+
+class App
+  attr_accessor :user_input
+
+  def initialize
+    puts 'Welcome to the Catalog of Things App!'
+    prompt
+    @books = []
+    @music_album = []
+    @movies = []
+    @games = []
+    @genre = []
+    @labels = []
+    @authors = []
+    @user_input = gets.chomp
+  end
+
+  def prompt
+    puts '
+    1 List all books
+    2 List all music albums
+    3 List all movies
+    4 List of games
+    5 List all genres
+    6 List all labels
+    7 List all authors
+    8 List all sources
```

Comment:
Hello @daudi13 we do not have a fourth partner so we won't be needing sources

Reply:
noted I will remove it from the file. Thank you :+1: